CMakeLists.txt

```diff
@@ -13,14 +13,16 @@ catkin_package()
 if (CATKIN_ENABLE_TESTING)
   find_package(roslaunch REQUIRED)
   roslaunch_add_file_check(tests/model_generators_roslaunch_test.xml)
+  find_package(rostest REQUIRED)
+  add_rostest(tests/model_generators_roslaunch_test.xml)
```

Comment:
we plan to do this repo ROS-agnostic, actually I am using it as a python module installed with the "pip" command.
The reason is that it has to work for ROS1 and ROS2, then probably these lines will be removed soon.

Reply:
same for me. I also use it as a python module. As you can find in [gitlab]( https://gitlab.cc-asp.fraunhofer.de/ipa326/ambs_creator/-/blob/main/install.sh)

I updated the example